### ISSUE TITLE
Change the title to romaji?

### ISSUE THREAD

**Issue by dhana-exe** [dmMaze/BallonsTranslator#214]
hello
umm i selected english as display language of the app....
but somehow on the app... the language are shown like this
![image](https://github.com/dmMaze/BallonsTranslator/assets/29838157/817df467-09e6-4fc7-a2a9-8c242c405eb3)
can somehow you on future update it to be shown as "romaji" or "latin" text?

i don't know which is japan, which is kor, which is china
maybe if it's all "romaji" or latinized we will surely know because we can read it and not grasp what is what

**Comment by dmMaze** [comment]
It is a bit troublesome to localize these parameters' displayed names due to the implementation, so there is no guarantee it will be supported anytime soon. You could translate target language names reversely, such as "Japanese" to "日本語" using google translate for now.

**Comment by dhana-exe** [comment]
yeah i currenly doing that but for somehow some of other "services" the name somehow seen so much similar so the cons is i need to change tab multiple times to make sure that the language that i choose is correct

**Comment by dhana-exe** [comment]
> > yeah i currently doing that but for somehow some of other "services" the name somehow seen so much similar so the cons is i need to change tab multiple times to make sure that the language that i choose is correct
> 
> You can suggest author add or replace with flags

i think this will be a great idea....
or maybe not "replace"
but "add" 

so we can see the word with original text with the flag....so it become more clear
